src/index.ts

```diff
@@ -31,3 +36,11 @@ export {
   apiSyntaxToFriendlySyntax,
   friendlySyntaxToApiSyntax,
 };
+
+yargs(hideBin(process.argv))
```

Comment:
Just moving this to /bin/index.ts is left

Reply:
Updated.  Confirmed by 
```adriantam@C02F5002MD6P syntax-transformer % npx @openfga/syntax-transformer transform
syntax-transformer transform

Transform

Options:
  --version    Show version number                                     [boolean]
  --help       Show help                                               [boolean]
  --from       whether we want to transform from dsl or json
                                             [required] [choices: "dsl", "json"]
  --inputFile  Configuration file. It must be in DSL syntax. [string] [required]

Missing required arguments: from, inputFile
```